cmds/exp/hdparm/hdparm.go

```diff
@@ -13,46 +13,51 @@
 //
 // We also have no plans to support ata12. It's 2019.
 //
+// Unlike the standard hdparm command, we do not allow empty
+// arguments for commands requiring a password.
+//
 // Synopsis:
-//     hdparm [--security-unlock] [--user-master|--timeout] [device ...]
+//     hdparm [--identify] [--security-unlock] [--user-master|--timeout] [device ...]
+//
 package main
 
 import (
 	"flag"
 	"fmt"
 	"log"
-	"os"
 
 	"github.com/u-root/u-root/pkg/scuzz"
 )
 
 // Because all the verbs and options are switches, hence global,
 // we can have all the functions take one Disk as a parameter and
-// return an error. This simplifies other aspects of this program.
+// return a Response and error. This simplifies other aspects of this program.
 type op func(scuzz.Disk) error
 
 var (
-	verbose = flag.Bool("v", false, "verbose log")
-	debug   = func(string, ...interface{}) {}
-	unlock  = flag.String("security-unlock", "", "Unlock the drive")
-	master  = flag.Bool("user-master", false, "Unlock master (true) or user (false)")
-	timeout = flag.Uint("timeout", 15000, "Timeout for operations")
-	verbs   = map[string]op{
-		"--security-unlock": unlockop,
-	}
+	verbose  = flag.Bool("v", false, "verbose log")
+	debug    = func(string, ...interface{}) {}
+	unlock   = flag.String("security-unlock", "", "Unlock the drive")
+	identify = flag.Bool("i", false, "Get drive identifying information")
```

Comment:
the documentation says --identify, but this says -i?

Reply:
yeah, turns out the command is -i ... will fix. I thought it was identify before.

Reply:
done